lib/pengine/bundle.c

```diff
@@ -292,21 +292,19 @@ create_docker_resource(pe_resource_t *parent, pe__bundle_variant_data_t *data,
              * monitor the child independently
              */
             crm_create_nvpair_xml(xml_obj, NULL, "monitor_cmd", "/bin/true");
-        /* } else if(child && data->untrusted) {
-         * Support this use-case?
-         *
-         * The ability to have resources started/stopped by us, but
-         * unable to set attributes, etc.
-         *
-         * Arguably better to control API access this with ACLs like
-         * "normal" remote nodes
-         *
-         *     crm_create_nvpair_xml(xml_obj, NULL,
-         *                           "run_cmd",
-         *                           "/usr/libexec/pacemaker/pacemaker-execd");
-         *     crm_create_nvpair_xml(xml_obj, NULL, "monitor_cmd",
-         *         "/usr/libexec/pacemaker/lrmd_internal_ctl -c poke");
+#if 0
```

Comment:
Now that these blocks are real code, it could make sense to use something that could be defined, so we can verify that it continues to compile.  Something like what we're doing with BUILD_PUBLIC_LIBPACEMAKER.

Reply:
It wouldn't compile, data->untrusted isn't a real thing. It's basically just a hint as to how such a thing might be implemented, if we ever want to. I doubt we ever will, and I considered just removing it, but I didn't want to tempt fate. :) This also needs some code de-duplication, but I didn't want to do that in a documentation PR, and I'm not sure when we'll get to it.